testing/test-commit.sh

```diff
@@ -26,17 +26,19 @@ commit="HEAD"
 parent=$(git log --pretty=%P -n 1 $commit)
 
 if grep -q " " <<< "$parent"; then
-    commit=$(cut -d" " -f2 <<< "$parent")
-    echo "HEAD is a merge commit. Taking the 2nd parent from $parent"
+    commit=$(sed 's/ /../g' <<< "$parent")
```

Comment:
I don't understand, isn't HEAD be the tip of the PR branch? In that case, why would that be a merge commit? Usually we expect PRs to be cleanly rebased so I woun't expect HEAD to be a merge commit

Reply:
no, I learned that when showing HEAD from PRs, the repo you get is a merge of `master` and `PR`, so that if your PR is based on an outdated `master`, you still test it as it would be after the PR is merged

that's also why PRs which `need-rebase` cannot be tested anymore, there's a conflict preventing the merge and thus github cannot prepare the commit